### PR TITLE
Please review the entire Question.java file with special attention to…

### PR DESCRIPTION
**Pull request by esengoba**
Look at these methods: setCategoryPathName() and selectQuestion()

### CODE REVIEW

File: src/model/Question.java [hunk 1]
```diff
@@ -141,35 +141,29 @@ public void setCategoryPathName(int path) {
 
      }
 
-    /**This method randomly generates a question ID number*/
+    /**This method randomly generates a question ID number.*/
      public int selectQuestion(){
          Random random = new Random();
 
          if (categoryID == 1) { //Geek Out
```
Comment (fmjosuel) [inline src/model/Question.java]: Isn't a switch statement suitable here? Instead of if/else's?

File: src/model/Question.java [hunk 2]
```diff
@@ -141,35 +141,29 @@ public void setCategoryPathName(int path) {
...
-             range = (int )(Math.random() * (120 - 101) + 101);
-             System.out.println("hi1 " +range);
+             randomID = (int )(Math.random() * (120 - 101) + 101);
```
Comment (fmjosuel) [inline src/model/Question.java]: `randomID = rand.nextInt(120) + 101;`
can alternatively do the job here. Kinda looks good, in my eyes!

File: src/model/Question.java [hunk 3]
```diff
@@ -141,35 +141,29 @@ public void setCategoryPathName(int path) {
...
-             System.out.println("hi5 " +range);
+             randomID = (int )(Math.random() * (505 - 501) + 501);
 
          } else if(categoryID == 6){ //Random
-             range = (int )(Math.random() * (605 - 601) + 601);
-             System.out.println("hi6 " +range);
+             randomID = (int )(Math.random() * (605 - 601) + 601);
          }
-         return range;
+         return randomID;
```
Comment (fmjosuel) [inline src/model/Question.java:168]: Also, can't we make these numbers dynamic? Say for example we decide to add questions in each category, we'd need to come back and change each number here, which is costly, in my opinion.

Comment (fmjosuel) [reply src/model/Question.java:168]: But otherwise this does the intended job!

**Review by fmjosuel — COMMENTED** [review]
Just a few comments!

File: src/model/Question.java (hunk 1 above)
Comment (esengoba) [reply src/model/Question.java]: Addressed in new pull request.

File: src/model/Question.java (hunk 2 above)
Comment (esengoba) [reply src/model/Question.java]: // We'll address this in a later pull request -- will require refactoring later to capture the number of questions in a given file

**Pull request merged**